solutions/cses-1619.mdx

```diff
@@ -184,6 +188,70 @@ int main(){
 }
 ```
 
+</CPPSection>
+	
+<JavaSection>
+	
+<Warning>
+Java solutions using this implementation will TLE on test case #6.
+</Warning>
+
+```java
+import java.io.*;
+import java.util.*;
+
+public class RestaurantCustomers {
+	static class CustomerData {
+		int time;
+		int type;
+	}
+
+	static class comparator implements Comparator<CustomerData> {
```

Comment:
Change to CustomerComparator so you can maintain upper camel case convention

Reply:
Also, you don't really need a custom class for this. To sort the array just do `Arrays.sort(data, Comparator.comparingInt((data) -> data.time));`

Reply:
doing that will make it less readable?

Reply:
Why? It makes it a lot simpler and removes the need for another comparator class, which is more confusing imo. Up to you tho

Reply:
If you do use the comparator though can you name it `CustomerComparator` because it's a class?

Reply:
it might be simpler, but usually people use comparators, very few know tricks like 

'
Arrays.sort(data, Comparator.comparingInt((customer) -> customer.time));
'